### DISCUSSION TITLE
I can't find the Cellar folder on my mac

### DISCUSSION THREAD

**Discussion by Nihilism0** [Homebrew/discussions#4298] [Everyday usage] [answered]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.0.4-192-gf02aea0
ORIGIN: https://mirrors.ustc.edu.cn/brew.git
HEAD: f02aea0238f06d432847daeaa37b1b3e459a070a
Last commit: 4 hours ago
Core tap origin: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 2e079663dddaa5552cb7ec56b77fafb53328dedb
Core tap last commit: 2 hours ago
Core tap branch: master
Core tap JSON: 04 Mar 03:22 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_BOTTLE_DOMAIN: https://mirrors.ustc.edu.cn/homebrew-bottles
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 10
Homebrew Ruby: 2.6.10 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: 10-core 64-bit arm_firestorm_icestorm
Clang: 14.0.0 build 1400
Git: 2.37.1 => /Applications/Xcode.app/Contents/Developer/usr/bin/git
Curl: 7.86.0 => /usr/bin/curl
macOS: 13.2.1-arm64
CLT: 14.2.0.0.1.1668646533
Xcode: 14.2
Rosetta 2: false
```

### Output of `brew doctor`

```shell
This command generates many files, which is too difficult to copy
```

### Description of issue

II can't find the Cellar folder in my finder. I need to copy some files in it to my GOPATH directory
<img width="588" alt="截屏2023-03-04 12 23 14" src="https://user-images.githubusercontent.com/114405451/222875526-f19ea527-5591-4840-986b-8ae62741ae56.png">
<img width="906" alt="截屏2023-03-04 12 24 35" src="https://user-images.githubusercontent.com/114405451/222875529-64e92fd9-54af-47e0-88fd-502926d54dec.png">

How can I access this file

**Comment by alebcay** [comment, marked as answer]
In Finder, press <kbd>⌘</kbd><kbd>⇧</kbd><kbd>G</kbd>. This should open a dialog box. You can enter the folder path `/opt/homebrew/Cellar` to access the contents of the Cellar.

**Reply by Nihilism0** [reply]
Thank you very much, I learned!!!

**Reply by SpecialCharacter** [reply]
Path did not work for me (I am on a Mac).

**Reply by SMillerDev** [reply]
If you are on an intel based Mac it's `/usr/local/Cellar`
